app/src/main/java/org/mozilla/reference/browser/ext/Context.kt

```diff
@@ -23,3 +31,27 @@ val Context.components: Components
 
 fun Context.getPreferenceKey(@StringRes resourceId: Int): String =
     resources.getString(resourceId)
+
+/**
+ *  Shares content via [ACTION_SEND] intent.
+ *
+ * @param text the data to be shared  [EXTRA_TEXT]
+ * @param subject of the intent [EXTRA_TEXT]
+ * @return true it is able to share false otherwise.
+ */
+fun Context.share(text: String, subject: String = ""): Boolean {
+    return try {
+        Intent(ACTION_SEND).apply {
+            type = "text/plain"
+            putExtra(EXTRA_SUBJECT, subject)
+            putExtra(EXTRA_TEXT, text)
+            flags = FLAG_ACTIVITY_NEW_TASK
+        }.also { intent ->
+            startActivity(intent)
+        }
```

Comment:
nit: This makes use of Kotlin's language features but I wonder if this is really easier to read than ``startActivity(Intent(..))```

Reply:
What about this:
```kotlin
 val intent = Intent(ACTION_SEND).apply {
            type = "text/plain"
            putExtra(EXTRA_SUBJECT, subject)
            putExtra(EXTRA_TEXT, text)
            flags = FLAG_ACTIVITY_NEW_TASK
        }
        startActivity(intent)
```
Uses remove the duplicate use of intent and it's easy to read.